roles/atat.builder/tasks/main.yml

```diff
@@ -31,3 +31,16 @@
     src: /usr/bin/python3
     dest: /usr/bin/python
     state: link
+
+- name: Register latest kubectl version
+  uri:
+    url: https://storage.googleapis.com/kubernetes-release/release/stable.txt
+    timeout: 10
+    return_content: yes
+  register: kubectl_version_response
+
+- name: Install kubectl
+  get_url:
+    url: "https://storage.googleapis.com/kubernetes-release/release/{{ kubectl_version_response.content | trim }}/bin/linux/amd64/kubectl"
```

Comment:
Will this install the latest `kubectl` version? Do we need to match versions with the rest of the cluster?

Reply:
1. Yes it would, the current stable release version any way (1.11.... something?)
1. Not necessarily. It doesn't hurt to try to match the version, but the clients have been pretty backwards compatible so far. If you want to lock it down, you may want to put the client you want to use in Cloud Files or something like that... I'd be concerned that always trying to pull an older version might just stop working.